### DISCUSSION TITLE
delete default exporters from WEB interface

### DISCUSSION THREAD

**Discussion by avoskresenskiy** [akvorado/akvorado#372] [Q&A] [answered]
Hi.
How I can remove default (fake) exporters (4 items) from WEB interface to see only number of the real exporters count in HOME page?

Thx

**Comment by vincentbernat** [comment, marked as answer]
I suppose you did disable the demo exporters. The easiest way would be to drop the truncate the `exporters` table in ClickHouse with `TRUNCATE TABLE exporters`.

**Reply by avoskresenskiy** [reply]
Thank a lot, Vincent.
Now I can see only real counter of the exporters on HOME page.

note
You're right, I've disabled the demo exporters before by quick start guide.
